content/marketing/demand-gen/marketing_operations.md

```diff
@@ -103,6 +105,10 @@ The parameters that [HubSpot supports out of the box](https://knowledge.hubspot.
 First Touchpoint is a custom marketing field we have created in Hubspot.. This fields help us determine the first touches marketing has with a prospect. There are supporting
 [Workflows](https://app.hubspot.com/workflows/2762526/flow/9011999/edit) set up in HubSpot to capture the latest conversion information, and copy this information to the ‘First event’ fields if they are blank/ unknown.
 
+### Hubspot User Data Deletion (GDPR requests, etc.)
+
+Sometimes prospects and customers will ask us to delete their email address and data because they want it done, or as part of GDPR compliance. We take these requests seriously, and you can read more about how we process these requests over in [this section of the handbook](https://handbook.sourcegraph.com/support/permanently_deleting_user_data).
+
```

Comment:
@rebecca-rissinger this link should work if you change it to:

../../support/permanently_deleting_user_data.md